js/app.js

```diff
@@ -429,8 +430,18 @@ function exportZip() {
 
 // export json function
 function exportAllJson() {
```

Comment:
There's nothing calling this function anymore

Reply:
And rather than having 2 entire functions to do the same thing just print a different string to the console, I think it would make more sense to consolidate them and run them through some shared conditional logic.

Reply:
at the moment we still call exportAllJson in menubar.js if the user requests. Agreed though, we probably could get rid of it.  I'm pretty sure they only asked for a json for a doc, or a zip file with everything

Reply:
there we go try that